docs/features/CLI.md

```diff
@@ -187,7 +187,7 @@ prompt. Here you can enter information to generate images from text
 Here are the invoke> command that apply to txt2img:
 
 | Argument <img width="680" align="right"/> | Shortcut <img width="420" align="right"/> | Default <img width="480" align="right"/> | Description                                                                                                                                                                                                                                      |
-| ----------------------------------------- | ----------------------------------------- | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|-------------------------------------------| ----------------------------------------- | ---------------------------------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
what's the reason for those changes? I am asking since VSCode will reformat this section next time I touch the file 😅

Reply:
Feel free to undo that... I just did this in pycharm and I guess it wanted formatting to be different.